src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/GEOS_LandAssimGridComp.F90

```diff
@@ -2338,7 +2278,129 @@ subroutine OUTPUT_SMAPL4SMLMC(gc, import, export, clock, rc)
      _RETURN(_SUCCESS)
 
   end subroutine OUTPUT_SMAPL4SMLMC 
- 
+
+  subroutine OUTPUT_INCR( cat_progn_incr, export,rc)
```

Comment:
If I'm reading this correctly, the subroutine output_incr() essentially copies data from the legacy LDASsa "cat_progn" structure into the MAPL export variables. 
I suggest renaming the subroutine (also to avoid confusion with the variable name "output_incr" above).
Maybe output_incr() could be rename to
   export_incr()
or 
   catprogn2export_incr()
or something else along those lines (avoiding the term "output" -- see comment above).

Reply:
Since it is private to the module, the subroutine name Export_INCR should be enough